sources/packages/backend/apps/api/src/route-controllers/report/_tests_/e2e/report.aest.controller.exportReport.e2e-spec.ts

```diff
@@ -249,4 +256,850 @@ describe("ReportAestController(e2e)-exportReport", () => {
+              addDays(1, fakeOfferingPartTime1.studyStartDate),
+            ),
+            studyEndDate: getISODateOnlyString(
+              addDays(1, fakeOfferingPartTime1.studyEndDate),
+            ),
+          },
+          programInitialValues: {
+            programIntensity: ProgramIntensity.fullTimePartTime,
+            deliveredOnSite: true,
+          },
+        },
+      );
+
+      // Create an education program with full time program intensity without any offering.
+      const fakeProgram = createFakeEducationProgram(
+        {
+          auditUser,
+          institution: institutionLocation.institution,
+        },
+        {
+          initialValues: {
+            programIntensity: ProgramIntensity.fullTime,
+            deliveredOnSite: true,
+          },
+        },
+      );
+      await db.educationProgram.save(fakeProgram);
+      await db.educationProgramOffering.save([
+        fakeOfferingFullTime1,
+        fakeOfferingFullTime2,
+        fakeOfferingPartTime1,
+        fakeOfferingPartTime2,
+      ]);
+
+      const ProgramAndOfferingStatusReport =
+        "Program_And_Offering_Status_Report";
+      const endpoint = "/aest/report";
+      const ministryUserToken = await getAESTToken(
+        AESTGroups.BusinessAdministrators,
+      );
+
+      // Payload with only full time option being checked.
+      const payloadFullTimeOnly = {
+        reportName: ProgramAndOfferingStatusReport,
+        params: {
+          institution: institutionLocation.institution.id,
+          startDate: fakeOfferingFullTime1.studyStartDate,
+          endDate: getISODateOnlyString(fakeOfferingPartTime2.studyEndDate),
+          offeringIntensity: {
+            "Full Time": true,
+            "Part Time": false,
+          },
+          sabcProgramCode: "",
+        },
+      };
+      const dryRunSubmissionMockFullTimeOnly = jest.fn().mockResolvedValue({
+        valid: true,
+        formName: FormNames.ExportFinancialReports,
+        data: { data: payloadFullTimeOnly },
+      });
+      formService.dryRunSubmission = dryRunSubmissionMockFullTimeOnly;
+
+      // Act/Assert
+      await request(app.getHttpServer())
+        .post(endpoint)
+        .send(payloadFullTimeOnly)
+        .auth(ministryUserToken, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.CREATED)
+        .then((response) => {
+          const fileContent = response.request.res["text"];
+          const parsedResult = parse(fileContent, {
+            header: true,
+          });
+          expect(parsedResult.data).toEqual(
+            expect.arrayContaining([
+              offeringToResultData(fakeOfferingFullTime1),
+              programToResultData(fakeProgram, institutionLocation),
+            ]),
+          );
+        });
+    },
+  );
+
+  it(
+    "Should generate the Program and Offering Status report when a report generation is made with the appropriate date range filters and only the part time option is considered " +
```

Comment:
The description states "and only the part time option is considered" but the API call has the below.
Is it the expectation?
```ts
offeringIntensity: {
  "Full Time": true,
  "Part Time": true,
},
```

Reply:
It should be "and both full time and part time options are considered ". It's now changed.